enterprise_access/apps/api_client/lms_client.py

```diff
@@ -115,6 +129,66 @@ def unlink_users_from_enterprise(self, enterprise_customer_uuid, user_emails, is
             logger.exception(msg, enterprise_customer_uuid)
             raise
 
+    def fetch_group_members(
+        self,
+        group_uuid,
+        sort_by=None,
+        user_query=None,
+        fetch_removed=False,
+        is_reversed=False,
+        traverse_pagination=False,
+        page=1,
+    ):
+        """
+        Fetches enterprise group member records from edx-platform.
+
+        Params:
+            - ``group_uuid`` (string, UUID): The group record PK to fetch members from.
+            - ``sort_by`` (string, optional): Specify how the returned members should be ordered. Supported sorting
+            values
+            are `member_details`, `member_status`, and `recent_action`.
+            - ``user_query`` (string, optional): Filter the returned members by user email with a provided sub-string.
+            - ``fetch_removed`` (bool, optional): Include removed membership records.
+            - ``is_reversed`` (bool, optional): Reverse the order of the returned members.
+            - ``traverse_pagination`` (bool, optional): Indicates that the lms client should traverse and fetch all
+            pages.
+            of data. Cannot be supplied if ``page`` is supplied.
+            - ``page`` (int, optional): Which page of paginated data to return. Cannot be supplied if
+            ``traverse_pagination`` is supplied.
+        """
+        if traverse_pagination and page:
+            raise FetchGroupMembersConflictingParamsException(
+                'Params `traverse_pagination` and `page` are in conflict, only supply one or the other'
+            )
```

Comment:
Hmm, maybe you want to copy this type of validation into the request serializer?